internal/controller/api/v1alpha1/image_types.go

```diff
@@ -20,22 +20,27 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
-// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
-
 // ImageSpec defines the desired state of Image
 type ImageSpec struct {
-	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
-	// Important: Run "make" to regenerate code after modifying this file
+	clusterName           string `json:"cluster_name,omitempty"`
```

Comment:
Should use ObjectReference instead of string.  See here for usage and definition:
https://github.com/kubernetes/api/blob/fcac651617f2c6b75a129dc2dcbdfd3aa8b0aa1c/core/v1/types.go#L5372

Reply:
will do.